shared/src/main/java/dice/StatsProcessor.java

```diff
@@ -161,42 +161,44 @@ private StatsData computeStats(RollRecord[] records) {
         double avgRolls = 0.0;
         double avgScore = 0.0;
         double avgNumDiceUsed = 0.0;
-
-        for (int index = 0; index < dbLength; index++) {
-            int temp = 0;
-            RollRecord record = records[index];
-            totalGames = record.getGameId();
-            totalDiceUsed += record.getNumDice();
-        }
-
-        avgNumDiceUsed = (double)totalDiceUsed / (double)dbLength;
-        totalGames = totalGames + 1;
-        avgRolls = (double)totalRolls / (double)totalGames;
-
         int maxGameScore = 0;
-        if (dbLength > 1) {
-            for (int index = 1; index < dbLength; index++) {
-                int[] tempArray = new int[totalGames];
+
+        if (dbLength != 0) {
```

Comment:
Can you test if it's failing on your end? I just tried on my laptop, just pulling from upstream master (no code from this commit), and TestDatabase and TestDiceGame are still failing.